Userland/Applications/MouseSettings/main.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, Idan Horowitz <idan.horowitz@gmail.com>
+ * Copyright (c) 2020-2021, the SerenityOS developers.
```

Comment:
Kinda gauche to remove someone else's copyright, I think you meant to add The SerenityOs developers 2021 below this like?

Reply:
Totally - I did not mean to stomp over that!

Reply:
Updated the license files so credit is given where due.

Reply:
Please drop the trailing commas though, this is how it's suppose to look:

https://github.com/SerenityOS/serenity/blob/0a911178cea4fcfb462eb7e256c51ed75756c7c8/Userland/Libraries/LibJS/Runtime/ArrayPrototype.cpp#L2-L4

Reply:
What about the single trailing dot at the end of "the SerenityOS deverlopers."? I've noticed that it's present most of the time.

Reply:
Dropped the dot as well, it's now following this style:
https://github.com/SerenityOS/serenity/blob/0a911178cea4fcfb462eb7e256c51ed75756c7c8/Userland/Libraries/LibCompress/Deflate.h#L2-L3